### PR TITLE
dyth conda

### PR DESCRIPTION
**Pull request by dyth**
Add missing package `blosc` to environment.yaml
Update documentation for installation with conda

### CODE REVIEW

File: environment.yaml [hunk 1]
```diff
@@ -9,4 +9,5 @@ dependencies:
     - pyqt
     - numpy
     - pip:
-        - gym
\ No newline at end of file
+        - gym
+        - blosc
```
[the remark this answers is not in the document]
Comment (maximecb) [reply environment.yaml]: Don't move the blosc dep out of pip... Some of use aren't using conda.

[the remark this answers is not in the document]
Comment (rizar) [reply environment.yaml]: But those who don't use Conda don't need `environment.yaml`, do they?

[the remark this answers is not in the document]
Comment (maximecb) [reply environment.yaml]: Ah there’s a pip section in the conda file. Carry on.

**Comment by dyth** [comment]
Found an additional "problem" in the instruction -- need to create environment, create module for gym_minigrid before creating module for babyai

File: README.md [hunk 2]
```diff
@@ -40,10 +40,27 @@ pip3 install --process-dependency-links --editable .
...
+After that, execute the following commands to setup the environment.
+
+```
+cd ..
+git clone https://github.com/maximecb/gym-minigrid.git
+cd gym-minigrid
+pip install -e .
+```
+
+The last command installs the repository as a module. Move back to the `babyai` repository and install that as a module.
```
Comment (rizar) [inline README.md]: I am not sure that "install as a module" is something people actually say. I think this kind of installation is called "install in the development mode"

Comment (rizar) [reply README.md]: From `pip install --help`:

` -e, --editable <path/url>   Install a project in editable mode (i.e. setuptools "develop mode") from a local project path or a VCS url.`

**Pull request merged**